### ISSUE TITLE
Please add remote server shell requirements to docs or FAQ

### ISSUE THREAD

**Issue by vvaves** [bit-team/backintime#1082]
I was trying to get back in time running pointing to my freenas server but it wouldnt create the folder on the remote server. Ended up just having to change the ssh user's shell to bash and everything worked! default on freenas is csh shell.

Thanks all working great!

**Comment by buhtz** [comment]
I wonder if we can do more about it then just documentation.

Question we should work on
 1. Can an ssh client force a specific shell on server side? (ask some admins about the "unix way")
 2. Minimally the shell type on server should be checked and an error should be thrown.

**Comment by buhtz** [comment]
I don't know much about csh shell. Can you give me an example which commands used by BIT do fail on a csh shell? I can not imagine one. To my understanding regular commands like `command --switch argument` should work on bash/sh and csh. And to my knowledge BIT doesn't use more complex shell commands.

I think about if BIT should check the remote shell offered via SSH and using an allow- and/or deny-list of shells. Or we should force the shell to use with adding extra commands (`bash -l`) when using `ssh` commands.

**Comment by emtiu** [comment]
> Can an ssh client force a specific shell on server side? (ask some admins about the "unix way")

I don't think so. You might run `bash` once you are logged in, but specifically on a FreeNAS (=FreeBSD) machine, it's possible that bash is not even installed.

> Can you give me an example which commands used by BIT do fail on a csh shell?

I don't have a good idea, either. I'm somewhat familiar with FreeBSD and use its default shell `csh` sometimes, but I'm unsure about this problem. Something along the lines of `$(command)` substitution or environmental variables, maybe.

**Comment by buhtz** [comment]
I have to close the issue if there not feedback from the opener.

I need an example as I asked before.

> Can you give me an example which commands used by BIT do fail on a csh shell? I can not imagine one. To my understanding regular commands like command --switch argument should work on bash/sh and csh. And to my knowledge BIT doesn't use more complex shell commands.

**Comment by buhtz** [comment]
Closing this ticket based on the comment above. Feel free to reopen
if the problem still exists. Thank you for your efforts. If you have
any further questions, ideas or encounter any other issues, please
don't hesitate to let us know.

Best regards,
